### PR TITLE
Add Singularity container definition and update Dockerfile

### PR DESCRIPTION
**Pull request by davidlougheed**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-# Run the model service server
-# provide some explicit defaults if no arugments are given
-ENTRYPOINT [ "python_model_service", "--port", "3000"]
-CMD [ "--logfile", "rnaget_service.log",\
-      "--database", "rnaget_service.sqlite" ]
+# Run the RNAget service server
+# provide some explicit defaults if no arguments are given
+ENTRYPOINT [ "candig_rnaget", "--port", "3000"]
+CMD [ "--logfile", "log/rnaget.log",\
+      "--database", "data/rnaget.sqlite" ]
```
Comment (alipski) [inline Dockerfile]: could you add "--tmpdata", "data/tmp/" to this entrypoint ?

File: rnaget.def [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    export HOME="/app"
+    ls /usr/local/lib
+    echo export PYTHONPATH=$PYTHONPATH:/usr/local/lib/python3.7/site-packages
+    cd /app
+    apt update && apt install -y libhdf5-dev
+    pip3 install --no-cache-dir -r requirements.txt
+    python3 setup.py install
+
+%startscript
+    exec candig_rnaget --port 3000 --logfile log/rnaget.log --database data/rnaget.sqlite
```
Comment (alipski) [inline rnaget.def]: to the exec, could you add --tmpdata data/tmp/ ?

Comment (davidlougheed) [reply rnaget.def]: d385342

File: Dockerfile (hunk 1 above)
Comment (davidlougheed) [reply Dockerfile]: d385342

**Pull request merged**